pkg/cmd/cli/cmd/importer/dockercompose.go

```diff
@@ -74,7 +74,7 @@ func NewCmdDockerCompose(fullName string, f *clientcmd.Factory, in io.Reader, ou
 	}
 	cmd := &cobra.Command{
 		Use:     "docker-compose -f COMPOSEFILE",
-		Short:   "Import a docker-compose.yml project into OpenShift",
+		Short:   "Import a docker-compose.yml project into OpenShift [EXPERIMENTAL]",
```

Comment:
I'm ambivalent about this - it ruins the look of the help page and makes the command more obvious (possibly not what we want).  Not sure I like the square brackets either.  Let me sleep on it.